torch/fx/experimental/proxy_tensor.py

```diff
@@ -120,13 +120,9 @@ def set_meta(proxy, val):
             proxy.node.meta['tensor_meta'] = _extract_tensor_metadata(val)
             # NB: Kinda hacky, but we should try to get val as the metadata
             # everywhere
-            try:
-                fake_tensor_mode = FakeTensorMode(allow_fallback_kernels=True)
-                proxy.node.meta['val'] = fake_tensor_mode.from_tensor(val)
-            except UnsupportedFakeTensorException:
-                pass
-            except NotImplementedError:  # FIXME: Happens with efficientzerotensor
-                pass
+            fake_tensor_mode = FakeTensorMode(allow_fallback_kernels=True)
+            with fake_tensor_mode:
+                proxy.node.meta['val'] = torch.empty_strided(val.shape, val.stride(), device=val.device, dtype=val.dtype)
```

Comment:
Storage offset ?

Reply:
Technically more correct but it's kind of annoying to create a tensor with a specified storage offset (and this is more of a hack/convenience feature than something we should be relying on anyways).